### PR TITLE
Made some changes to help with flow for native English speaker

### PR DESCRIPTION
**Pull request by dholbrook86**
Changed a bit of text to help with the sentence flow that a native English speaker would be accustomed to. Minor changes. Your English is awesome! This PR will just help it read much better.

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -41,48 +41,48 @@ function IndexPage() {
                 <div className="bg-gray-900 rounded-lg px-3 py-20 md:col-span-1">
                     <div className="text-center mb-5 mx-auto">
                         <h1 className="font-bold text-white text-3xl mt-4">Your masterpiece,<br />one tap away.</h1>
-                        <p className="text-gray-300">Access all the power of Kontax Cam with just a tap away.</p>
+                        <p className="text-gray-300">Access all the power of Kontax Cam with just a tap.</p>
```
Comment (dholbrook86) [inline src/pages/index.js:44]: You don't need the word "Away". Sounds repetitive from the sentence above it.

File: src/pages/index.js [hunk 2]
```diff
@@ -41,48 +41,48 @@ function IndexPage() {
...
                     </div>
                     <img className="w-10/12 h-auto mx-auto" src={image1} alt="Kontax Cam User interface" />
                 </div>
 
                 <div id="features" className="bg-gray-900 rounded-lg px-10 py-20 md:col-span-2">
                     <div className="text-center">
                         <h1 className="font-bold text-3xl mt-4">Carefully crafted effects</h1>
-                        <p className="text-gray-300">All of these handcrafted effects, plus more in the future!</p>
+                        <p className="text-gray-300">All of these handcrafted effects, plus more coming soon!</p>
```
Comment (dholbrook86) [inline src/pages/index.js:52]: Coming soon is much more common to say

File: src/pages/index.js [hunk 3]
```diff
@@ -41,48 +41,48 @@ function IndexPage() {
...
                     </div>
 
                     <div className="grid gap-4 grid-cols-1 md:grid-cols-2 mt-20">
                         {
                             [{
                                 icon: colorIcon,
                                 title: `Colour leaks`,
-                                description: `Colour leaks add a gorgeous film colour overlay into your photo. 
+                                description: `Colour leaks add a gorgeous film colour overlay on your photo.
```
Comment (dholbrook86) [inline src/pages/index.js:60]: You wouldn't put effects inside of a photo, you would put the effects on them. So I made this change.

File: src/pages/index.js [hunk 4]
```diff
@@ -41,48 +41,48 @@ function IndexPage() {
...
                                 it more unique.`,
                             },
                             {
                                 icon: grainIcon,
                                 title: `Film grain`,
-                                description: `Grain makes your photo looks sharp, scratched and old school!
-                                Although ecessive use might render the photo too over-sharpen,
-                                grain is beloved in the heart of film photographers.`,
+                                description: `Grain makes your photo look sharp, scratched, and old school!
+                                Although excessive use might render the photo overly shapened.
```
Comment (dholbrook86) [inline src/pages/index.js]: I made a mistake here. The word should be "sharpened" not "shapened".

File: src/pages/index.js [hunk 5]
```diff
@@ -41,48 +41,48 @@ function IndexPage() {
...
+                                Film photographers love this effect.`,
                             },
                             {
                                 icon: leakIcon,
                                 title: `Light leaks`,
-                                description: `Light leaks make your photo pop with custom made film light leaks.
+                                description: `Light leaks make your photo pop with custom made film light effects.
```
Comment (dholbrook86) [inline src/pages/index.js]: Saying "leaks" again is redundant

**Comment by kxvn-lx** [comment]
Thank you so much Darin! I really appreciate your help.

**Comment by dholbrook86** [comment]
You’re welcome Kevin!

Connect with me on LinkedIn, you never know when the connection will come
in handy. No strings attached. I’m not trying to peddle anything. I just
like connecting.

http://linkedin.com/in/darinholbrook

**Comment by kxvn-lx** [comment]
Of course! just added. 👍

**Pull request merged**